src/main/java/com/google/sps/servlets/RoutesStoring.java

```diff
@@ -44,6 +44,22 @@ String getErrorMessage() {
     }
   }
 
+  public enum UserAccessType {
+    OWNER(1),
```

Comment:
It is customary to start enums from 0, with 0 either being the default value, or a 'NOT_SET' type value.